src/batch.rs

```diff
@@ -431,6 +432,11 @@ fn ddbjson_val_to_attrval(ddb_jsonval: &JsonValue) -> Option<AttributeValue> {
             n: Some(x.as_str().unwrap().to_string()),
             ..Default::default()
         })
+    } else if let Some(x) = ddb_jsonval.get("B") {
+        Some(AttributeValue {
+            b: Some(Bytes::copy_from_slice(x.as_str().unwrap().as_bytes())),
```

Comment:
Based on the test code, we should pass the base64 decoded data into `Bytes::copy_from_slice` or a similar method.